apps/frontend/src/libs/components/pagination/libs/helpers/get-pages-cut.helper.ts

```diff
@@ -0,0 +1,40 @@
+type GetPagesCutParameters = {
+	currentPage: number;
+	pagesCount: number;
+	pagesCutCount: number;
+};
+
+type GetPagesCutReturn = {
+	end: number;
+	start: number;
+};
+
+const ONE_ITEM = 1;
+const HALF_DIVIDER = 2;
+
+const getPagesCut = ({
```

Comment:
```suggestion
const getPagesCut: FnType = ({
```
combine GetPagesCutParameters and GetPagesCutReturn into one type